ERCS/erc-7891.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 7891
+title: Splitting and Merging of NFTs
+description: Hierarchical NFTs with Splitting and Merging
```

Comment:
Your description mostly restates your title. It would be better to use this space to define "hierarchical", "splitting", and "merging".